### PR TITLE
New User Email Notification

### PR DESCRIPTION
**Pull request by dylanj1752**
## 🗣 Description ##

Add Env variables for new user registration email notification

## 💭 Motivation and context ##

Allow the admins to be notified when a new user has registered with the application.

## ✅ Checklist ##

* [x] This PR has an informative and human-readable title.
* [x] Changes are limited to a single goal - _eschew scope creep!_
* [x] All relevant type-of-change labels have been added.
* [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
* [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).

### CODE REVIEW

**Review by zenine07 — CHANGES_REQUESTED** [review]
I see you put the `"NEW_USER_NOTIFICATION_EMAIL_ADDRESS": var.new_user_notification_email_address` in `cool/api.tf`, but I'm not seeing that env var under the one in `terraform/locals.tf`, which is our INL sandbox environment. I thought we added it when working together, just not seeing it here.

File: terraform/vars.tf [hunk 1]
```diff
@@ -13,6 +13,13 @@ variable "region" {
   type = string
 }
 
+#=================================================
+#  EMAILS NOTIFICATIONS
+#=================================================
+variable "new_user_notification_email_address" {
+  type = string
+}
+
```
Comment (zenine07) [inline terraform/vars.tf]: No seeing a similar snippet in `cool/vars.tf` as well.

File: terraform/vars.tf [hunk 2]
```diff
@@ -13,6 +13,13 @@ variable "region" {
   type = string
 }
 
+#=================================================
+#  EMAILS NOTIFICATIONS
+#=================================================
+variable "new_user_notification_email_address" {
+  type = string
+}
```
Comment (jsf9k) [inline terraform/vars.tf]: All Terraform variables should have a `description` property.

**Review by jsf9k — CHANGES_REQUESTED** [review]
Please add `description` fields for all Terraform variables.

**Comment by jsf9k** [comment]
@dylanj1752, in the PR description the checkboxes should look like `[x]` when checked (no spaces).   GitHub Markdown renders them nicely when you do that.  (You can alternatively just click on the unchecked boxes and they will automatically change state.)

I made the change here already, but please try that going forward.

**Comment by dylanj1752** [comment]
> @dylanj1752, in the PR description the checkboxes should look like `[x]` when checked (no spaces). GitHub Markdown renders them nicely when you do that. (You can alternatively just click on the unchecked boxes and they will automatically change state.)
> 
> I made the change here already, but please try that going forward.

File: terraform/vars.tf (hunk 1 above)
Comment (dylanj1752) [reply terraform/vars.tf]: Resolved

**Review by zenine07 — APPROVED** [review]

File: terraform/vars.tf (hunk 2 above)
Comment (zenine07) [reply terraform/vars.tf]: @jsf9k - @dylanj1752 added those description fields, can you take a look at this again?

Comment (jsf9k) [reply terraform/vars.tf]: _All_ Terraform variables should have a `description` property, not just the one created in this PR.  That said, I'll go ahead and approve if someone creates an issue in this repo to add the `description` properties later; I just don't want it to be forgotten.

Comment (zenine07) [reply terraform/vars.tf]: @jsf9k I have created an issue https://github.com/cisagov/domain-manager-cicd/issues/9 and assigned it to myself.

**Review by jsf9k — APPROVED** [review]

**Pull request merged**